lighthouse-core/audits/predictive-perf.js

```diff
@@ -71,6 +72,34 @@ class PredictivePerf extends Audit {
     return scriptUrls;
   }
 
+  /**
+   * @param {!Node} dependencyGraph
+   * @return {!Object}
+   */
+  static computeOptions(dependencyGraph) {
```

Comment:
i appreciate the thought you put into the names in this file, but i think we can upgrade this one a bit. 
`computeRTTandSR` ?

Reply:
sg done